js/hang/src/watch/audio/source.ts

```diff
@@ -172,7 +172,9 @@ export class Source {
 		});
 		effect.cleanup(() => consumer.close());
 
-		let totalBytes = 0;
+		// Persist totalBytes across rendition changes
+		const currentStats = this.#stats.peek() ?? { bytesReceived: 0 };
+		let totalBytes = currentStats.bytesReceived;
```

Comment:
To be safer and support rendition switching (in the future), can you replace this local variable with:

```typescript
this.#stats.update((stats) => {
  bytesReceived: (stats?.bytesReceived ?? 0) + byteLength,
});
```

just like how video works